tests/base.py

```diff
@@ -196,6 +196,10 @@ def assertStatus(self, response, code):
         """
         code = str(code)
         msg = 'Response status was %s, not %s.' % (response.output_status, code)
+
+        if response.json:
+            msg += ' Response body was %s.' % json.dumps(response.json)
```

Comment:
@cjh1 I like this. We could go even prettier, though. `json.dumps(response.json, sort_keys=True, indent=4, separators=(',', ': '))`. :dancer: